docs/use_cases/model_applications/s2s_mjo/UserScript_obsCFSR_obsOnly_MJO_ENSO.py

```diff
@@ -81,24 +81,28 @@
 
 ##############################################################################
 # MET Configuration
-# ---------------------
-#
-# METplus sets environment variables based on the values in the METplus configuration file.
-# These variables are referenced in the MET configuration file. **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!** If there is a setting in the MET configuration file that is not controlled by an environment variable, you can add additional environment variables to be set only within the METplus environment using the [user_env_vars] section of the METplus configuration files. See the 'User Defined Config' section on the 'System Configuration' page of the METplus User's Guide for more information.
+# -----------------
 #
+# There are no MET tools used in this use case.
+  
+##############################################################################
+# Python Embedding
+# ----------------
 #
+# Python Embedding  is not used in this use case.
 
 ##############################################################################
-# Python Scripts
-# ----------------
+# User Scripting
+# --------------
 #
 # The MJO-ENSO driver script orchestrates the calculation of the MaKE and MaKI indices and 
 # the generation of a text file and a plot for the indices:
 # parm/use_cases/model_applications/s2s_mjo/UserScript_obsCFSR_obsOnly_MJO_ENSO/mjo_enso_driver.py:
 #
-# .. highlight:: python
-# .. literalinclude:: ../../../../parm/use_cases/model_applications/s2s_mjo/UserScript_obsCFSR_obsOnly_MJO_ENSO/mjo_enso_driver.py
+# .. dropdown::
```

Comment:
```suggestion
# .. dropdown:: mjo_enso_driver.py
```

Reply:
HUGE THANK YOU @jprestop.  I have had a day and a half and clearly wasn't that careful.  Thanks for catching these mistakes.